### PR TITLE
site: Fixup RateLimiting Guide & some other links

### PR DESCRIPTION
**Pull request by stevesloka**
Fixes up the ratelimiting guide's links along with some other
links in the SITE_CONTRIBUTION, and getting started guide.

Upates #3712

Signed-off-by: Steve Sloka <slokas@vmware.com>

### CODE REVIEW

File: site/content/guides/global-rate-limiting.md [hunk 1]
```diff
@@ -424,12 +424,12 @@ For more information, see the [Contour rate limiting documentation][7] and the [
...
 [3]: https://projectcontour.io/getting-started/#option-1-quickstart
 [4]: https://github.com/envoyproxy/ratelimit
 [5]: https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/ratelimit/v3/rls.proto
 [6]: https://github.com/envoyproxy/ratelimit#configuration
-[7]: /docs/{{site.latest}}/config/rate-limiting/
-[8]: /docs/{{site.latest}}/config/api/
-[9]: {{site.github.repository_url}}/tree/main/examples/ratelimit
+[7]: /docs/{{< param docs_latest >}}/config/rate-limiting/
+[8]: /docs/{{< param docs_latest >}}/config/api/
+[9]: {{< param github_base_url>}}/tree/main/examples/ratelimit
```
Comment (skriss) [inline site/content/guides/global-rate-limiting.md:434]:
```suggestion
[9]: {{< param github_url >}}/tree/main/examples/ratelimit
```

Comment (stevesloka) [reply site/content/guides/global-rate-limiting.md]: We had two for some reason, took out the extra.

Comment (skriss) [reply site/content/guides/global-rate-limiting.md]: Yeah just noticed that, 👍

File: SITE_CONTRIBUTION.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ Additionally, the above method will not follow any permalink changes or page-spe
 
 **Linking to the project contour GitHub repo:**
 
-The GitHub metadata plugin populates Jekyll variables with data from GitHub during build. When inserting a link to the Project Contour repository, use the prefix `{{site.github.repository_url}}`.
+The GitHub metadata plugin populates Jekyll variables with data from GitHub during build. When inserting a link to the Project Contour repository, use the prefix `{{< param github_url >}}`.
```
Comment (skriss) [inline SITE_CONTRIBUTION.md:92]: Fine for now but this page is completely inaccurate now. #3733

**Review by skriss — APPROVED** [review]

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**